pkg/binding/placement.go

```diff
@@ -124,6 +125,22 @@ func (c *Controller) getPlacementByName(name string) (runtime.Object, error) {
 	return got, nil
 }
 
+func (c *Controller) createBindingsFromBindingPolicies(ctx context.Context) {
```

Comment:
I think it is most accurate to call this `evaluateBindingPolicies` or `enqueueBindingPolicies` because this is what the function does. Beyond this point, the evaluation of bindingpolicies would indeed queue up bindings for syncing if changed, but would only create them if they don't exist.

If you accept this, the log text below and the TODO comment in controller::createManagedClustersInformer should be updated.

Reply:
Accepted and amended.